native/KotlinIOS/KotlinIOS/AdvancedSearchCellInformation.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+struct AdvancedSearchCellInformation {
+    var label: String
+    var dataType: cellDataType
```

Comment:
I would probably also put the definition of the cellDataType enum in here, since it's very heavily associated with this struct